__TESTS__/urlEncodeDecode.spec.tsx

```diff
@@ -10,19 +10,19 @@ describe('URLEncodeDecode', () => {
   it('should encode and decode', () => {
     render(<URLEncodeDecode />);
 
-    const urlInput = screen.getByTestId('URL') as HTMLInputElement;
```

Comment:
Using `getByLabelText` instead of `getByTestId` makes for a better test case because we're also making sure the inputs are accessible. Not a huge deal, feel free to leave as is.